openai/openai.go

```diff
@@ -367,6 +403,18 @@ func fromChatRequest(r ChatCompletionRequest) (*api.ChatRequest, error) {
 			}
 			messages = append(messages, message)
 		default:
+			if msg.Role == "assistant" && msg.ToolCalls != nil {
+				toolCalls := make([]api.ToolCall, len(msg.ToolCalls))
+				for i, tc := range msg.ToolCalls {
```

Comment:
I think we can just append the tools no matter what

Reply:
oh hm I wanted to keep it in the switch since mike said including content and tool calls in the same message is undefined, what do you think

Reply:
that's ok then